src/stories/HeatmapVis.stories.tsx

```diff
@@ -12,7 +12,9 @@ import { ScaleType } from '../h5web/visualizations/shared/models';
 import { INTERPOLATORS } from '../h5web/visualizations/heatmap/interpolators';
 
 // A 2D dataset
-const dataset = mockData.datasets['c8f60c29-aae2-11ea-84a9-b94ddd2ec9e8'];
+const dataset = Object.values(mockData.datasets).find(
+  (d) => d.alias[0] === '/nD/twoD'
+) as { value: number[][]; shape: { dims: number[] } };
```

Comment:
Could you maybe re-use an existing provider interface for the casting?

Reply:
If I use `HDF5...` types, I run into all sorts of TS errors: the JSON data appears to not fit exactly that and then, I still need to cast afterwards as the `HDF5...` are too generic.